ThunderCats/UITextField+TCAdditions.h

```diff
@@ -38,4 +38,26 @@
  */
 - (void)tc_setLeftAndRightPadding:(CGFloat)paddingWidth;
 
+/**
+ *  Adds a toolbar to keyboard with a next function which advances to the next textfield when tapped.
+ *
+ *  @param nextTextField The textfield that should becomeFirstResponder next
+ *  @param toolbarColor  The UIColor of the toolbar
+ */
+- (void)tc_addNextButtonToolbarWithColor:(UIColor * __tc_nonnull)toolbarColor
+                               nextField:(UITextField * __tc_nonnull)nextTextField;
+
+/**
```

Comment:
you are right, I refactored based on your and Logan's comments and did not rememeber to update the comments